elements/pfe-primary-detail/src/pfe-primary-detail.scss

```diff
@@ -0,0 +1,98 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+
+$LOCAL: primary-details;
+
+// This variable is global so that helper functions can reference it
+$LOCAL-VARIABLES: (
+  Border: 1px solid pfe-var(ui--border--lighter),
+
+  // Applied to the layout of the nav vs details
+  GridTemplateColumns: 1fr 2fr,
+
+  // Styles have important since they are slotted
+  nav: (
+    Color: #151515!important, // pfe-var(text)
```

Comment:
Does this work?  I think you need to put the `!important` in the property declaration and not in the variable.